gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/Format.feature

```diff
@@ -0,0 +1,141 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepFormat
+Feature: Step - format()
+
+  Scenario: g_VX1X_formatXstrX
```

Comment:
A few tests to consider:
1) Null argument.
2) Empty String argument.
3) A bigger mix of positional and named, so ${value}, ${_}, ${value2}, ${_} ...
4) A test with an unsupported traverser (like a list).